wren-ai-service/eval/metrics/column.py

```diff
@@ -11,33 +11,60 @@
 
 class AccuracyMetric(BaseMetric):
     def __init__(self, engine_config: dict):
-        self.threshold = 10
+        self.threshold = 0
         self.score = 0
         self._engine_config = engine_config
 
     def measure(self, test_case: LLMTestCase):
         return asyncio.run(self.a_measure(test_case))
 
     def is_subset(self, expected: pd.DataFrame, actual: pd.DataFrame) -> bool:
-        if not set(actual.columns).issubset(set(expected.columns)):
+        if not set(expected.columns).issubset(set(actual.columns)):
             return False
 
-        common_columns = actual.columns
-        if common_columns.empty:
+        common_columns = sorted(expected.columns)
+        if not common_columns:
```

Comment:
we use expected columns as the common. thus, we might consider changing the condition to check if the actual columns are empty. it seems unlikely that the expected columns will be empty. wdyt?

Reply:
ok, removed